DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -144,7 +144,7 @@ public List<CentreUserDetails> GetUserCentres(AdminUser? adminUser, List<Delegat
         public bool NewEmailAddressIsValid(string emailAddress, int? adminUserId, int? delegateUserId, int centreId)
         {
             var (adminUser, delegateUser) = GetUsersById(adminUserId, delegateUserId);
-            if (adminUser?.EmailAddress == emailAddress && delegateUser?.EmailAddress == emailAddress)
+            if ((adminUser == null || adminUser.EmailAddress == emailAddress) && (delegateUser == null || delegateUser.EmailAddress == emailAddress))

```

Comment:
This statement is getting quite long - maybe worth pulling out into a method that's called `userEmailHasChanged(user, email)` or similar and doing if `(!userEmailHasChanged(adminUser, emailAddress) && !userEmailHasChanged(delegateUser, emailAddress) `